sklearn/linear_model/tests/test_ridge.py

```diff
@@ -311,6 +311,72 @@ def test_ridge_individual_penalties():
     assert_raises(ValueError, ridge.fit, X, y)
 
 
+def test_ridge_gcv_vs_k_fold():
```

Comment:
This might simplify a bit the readibility

```python
@pytest.mark.parametrize('gcv_mode', ['svd', 'eigen'])
@pytest.mark.parametrize('X_constructor', [np.asarray, sp.csr_matrix])
@pytest.mark.parametrize('X_shape', [(11, 8), (11, 20)])
@pytest.mark.parametrize('fit_intercept', [True, False])
@pytest.mark.parametrize('normalize', [True, False])
def test_ridge_gcv_vs_k_fold(gcv_mode, X_constructor, X_shape, fit_intercept,
                             normalize):
    n_samples, n_features = X_shape
    X, y = make_regression(
        n_samples=n_samples, n_features=n_features, n_targets=3,
        random_state=0, shuffle=False, noise=30., n_informative=5
    )
    X += 30 * np.random.RandomState(0).randn(X.shape[1])

    alphas = [1e-3, .1, 1., 10., 1e3]
    loo_ridge = RidgeCV(cv=n_samples, fit_intercept=fit_intercept,
                        alphas=alphas, scoring='neg_mean_squared_error',
                        normalize=normalize)
    gcv_ridge = RidgeCV(gcv_mode=gcv_mode, fit_intercept=fit_intercept,
                        alphas=alphas, normalize=normalize)

    loo_ridge.fit(X, y)

    X_gcv = X_constructor(X)
    gcv_ridge.fit(X_gcv, y)

    assert gcv_ridge.alpha_ == pytest.approx(loo_ridge.alpha_)
    assert_allclose(gcv_ridge.coef_, loo_ridge.coef_, rtol=1e-4)
    assert_allclose(gcv_ridge.intercept_, loo_ridge.intercept_, rtol=1e-4)
```

Reply:
> This might simplify a bit the readibility

thanks!